reference/release-notes/1.25.0.md

```diff
@@ -0,0 +1,115 @@
+* Enhancements to the following forms and pages:
+    - *Create Application*
+    - *Edit Application*
+    - *Create Instance*
+    - *Profile*
+    - *Application details*
+
+### AMS
+* When querying image related information, you can now see the image variant without waiting for the full image to be downloaded.<!--AC-2987-->
+* The GPU type is included in the AMS node API object, to provide better visibility into hardware specifications.<!--AC-3004-->
+
+### Stream Gateway
+* An administrator can revoke a pre-signed ADB connection URL before it expires, to prevent malicious actions using the command:
+
+    anbox-stream-gateway share delete <share_id>
+
+See {ref}`howto-access-instance` for more information.<!--AC-3052-->
+* You can now update the expiration date, description and the timeout values of a shared session using the command:<!--AC-3053-->
+
+        anbox-stream-gateway share update <share_id> --expiry=30m --description=new_description
+* The Stream Gateway now uses the latest [Dqlite LTS version](https://dqlite.io/).<!--AC-2990-->
+* Anbox Cloud uses WebRTC 6778.<!--AC-2904-->
+
+### Monitoring
+* A new `/1.0/metrics` endpoint is added to support collecting metrics from the Anbox runtime.<!--AC-3062-->
+
+### Other
+* Android security updates for February 2025 (see [Android Security Bulletin - February 2025](https://source.android.com/docs/security/bulletin/2025-02-01) for more information).
+* The Android WebView has been updated to 132.0.6834.80.<!--AC-3105-->
+
+## Removed functionality
+
+* Starting with the 1.25.0 release, the appliance snap with `epoch=0` is removed and unsupported. This epoch will no longer receive updates.
+To start using the new appliance snap with `epoch=1`, a fresh install of the appliance snap is required.
+
+## Deprecations
+
+The following deprecations are announced as part of the 1.25.0 release. See {ref}`ref-deprecation-notes` for more information on when the support for these items will be dropped:
+
+* The support for LXD 5.0 snap is deprecated.<!--AC-2734-->
+* The support for kernel 6.8 and earlier is deprecated.<!--AC-3106-->
+
+## CVEs
+
+The Anbox Cloud 1.25.0 release includes fixes from the respective upstreams and components for the following CVEs:
+
+| CVE | Affected Components |
+|-----|---------------------|
+|CVE-2024-49721 | Android 12,13 |
+|CVE-2023-40133 | Android 12,13 |
+|CVE-2023-40134 | Android 12,13 |
+|CVE-2023-40135 | Android 12,13 |
+|CVE-2023-40136 | Android 12,13 |
+|CVE-2023-40137 | Android 12,13 |
+|CVE-2023-40138 | Android 12,13 |
+|CVE-2023-40139 | Android 12,13 |
+|CVE-2024-49743 | Android 12,13,14 |
+|CVE-2024-49746 | Android 12,13,14 |
+|CVE-2023-40122 | Android 12,13,14 |
+|CVE-2024-49729 | Android 12,13,14 |
+|CVE-2024-0037  | Android 12,13,14 |
+|CVE-2025-0100  | Android 12,13,14 |
+|CVE-2024-49741 | Android 12,13,14 |
+|CVE-2025-0094  | Android 12,13,14 |
+|CVE-2025-0091  | Android 12,13,14 |
+|CVE-2025-0095  | Android 14 |
```

Comment:
CVE-2024-53103 has been fixed in the latest kernel, I think we should add it here for VM images?